VirtualDevelopmentEnvironment.md

```diff
@@ -1,41 +1,50 @@
-## Virtual development environment
+# Virtual Development Environment
 
-It's recomended to use [Vagrant][] and [VirtualBox][] in order to achieve consistent development environment across all contributors.
+It's recomended to use [Vagrant][] and [VirtualBox][] in order to achieve
+consistent development environment across all contributors.
 
-### Installation
+## Installation
 
-- Download and install latest [VirtualBox][]
-- Download and install latest [Vagrant][]
-- Clone GitHub repo:
-    
-    ```
+1.  Download and install latest [VirtualBox][].
+2.  Download and install latest [Vagrant][].
+3.  Clone GitHub repo:
+
+    ```shell
     $ git clone git@github.com:apiaryio/dredd.git
```

Comment:
Good point. I took this from the original document. I always wondered why GitHub prefers HTTPs in their UI and docs and now it makes sense :-)